include/emp/web/Widget.hpp

```diff
@@ -48,6 +48,16 @@
 namespace emp {
 namespace web {
 
+  // To allow vectorized appending of components
```

Comment:
probably also need to detect `emp::vector`

Move these helpers into `meta/` and be sure to unit test them